### DISCUSSION TITLE
Various keys cannot be sent properly

### DISCUSSION THREAD

**Discussion by alexanderadam** [AppGround-io/bluetooth-keyboard-and-mouse-support#216] [Bugs & Problems]
Version: Serverless Bluetooth Keyboard & Mouse Premium 4.7.1

Tilde (`~`) is send as negation (`¬`) instead

And most characters in the first row of the keyboard in the screen aren't send at all (I think the pipe `|` was the last character that's working).
![Special Characters on GBoard in Firefox](https://user-images.githubusercontent.com/372620/149640186-a338f1a9-821d-4dba-a903-7e5a7b7944bc.png)

This is the standard Gboard keyboard.

It reminds me on [the backslash issue](https://github.com/AppGround-io/bluetooth-keyboard-and-mouse-support/discussions/189) but backslash is working fine for me.

**Comment by AppGround-io** [comment]
The app can only send keys that are available on the selected language layout, so only a small subset of Unicode characters are supported.

Regarding the problem with `~`: 
The layout do not match on the device with the app and the device you want to control. Please try to switch to English US in the settings of the app because it seems that your remote device is using the English keyboard layout.

**Reply by alexanderadam** [reply]
I made sure that everythin has US English now but I still cannot write tilde (it's great though, that the app is giving hints! 👍).
